tanjun/dependencies.py

```diff
@@ -381,9 +913,12 @@ def set_standard_dependencies(client: injecting.InjectorClient, /) -> None:
     client: tanjun.injecting.InjectorClient
         The injector client to set the standard dependencies on.
     """
+    InMemoryCooldownManager().add_to_client(client)
```

Comment:
Maybe don't set this as a cooldown by default since ig the assumption is that running this without any set buckets should almost always be erroneous?